scripts/verifyZkSyncDeployments.ts

```diff
@@ -0,0 +1,159 @@
+// zksync block explorer does not support reading and writing to implementation contracts via a proxy
+// contract yet, so verification of the contract configuration/state must be done programatically.
+// To verify the zkSync SpokePool:
+//   yarn hardhat run --network zksync ./scripts/verifyZkSyncDeployments.ts
+//
+// To verify the zkSync adapter on mainnet:
+//   yarn hardhat run --network mainnet ./scripts/verifyZkSyncDeployments.ts
+
+import { assert } from "console";
+import { ContractFactory } from "ethers";
+import { Provider as zkProvider, utils as zkUtils } from "zksync-web3";
+import { ethers, toBN, findArtifactFromPath } from "../utils/utils";
+
+const textPadding = 30;
+
+/**
+ * l2RefundAddress and spokePoolDeploymentBlocks can change per deployment.
+ */
+const l2RefundAddress = "0x9A8f92a830A5cB89a3816e3D267CB7791c16b04D";
```

Comment:
This should not be the dev wallet as the mnemonic is shared too widely we should use one of the gckms keys

Reply:
Great catch - that was a mistake. Have fixed it now :+1: